### PR TITLE
Added rollup config version

### PR DESCRIPTION
**Pull request by chimon2000**
This version reduces the file size further to 74kb minified (24 gzipped).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,16 @@
...
+    "rollup-plugin-buble": "^0.15.0",
+    "rollup-plugin-gzip": "^1.0.0",
+    "rollup-plugin-node-resolve": "^3.0.0",
+    "rollup-plugin-replace": "^1.1.1",
+    "rollup-plugin-uglify": "^1.0.1",
+    "rollup-plugin-vue": "^2.3.1",
     "uglify-js": "git://github.com/mishoo/UglifyJS2.git#harmony",
     "vue": "^2.2.6",
-    "vue-custom-element": "^1.0.13",
+    "vue-custom-element": "git+https://github.com/chimon2000/vue-custom-element.git",
```
Comment (kartsims) [inline package.json]: Why didn't you use NPM version ?

**Review by kartsims — COMMENTED** [review]
Thanks for the PR !

That's great that you could reduce the size furthermore. I didn't know Rollup and since it is not as popular as Webpack, I'll say we could put it in a separate branch.

So to get it merged, could you please :

- Remove all reference to Webpack
- Replace the NPM scripts to use rollup instead of webpack (the `dev` and `build` scripts)
- Explain why you are using your own fork of `vue-custom-element` rather than the original one

I am really glad that you could have got it below the 100kB, it is becoming rather acceptable...!

I hope to be able to merge this soon and update the post as I get many page views since Vue.js newsletter relayed the post and it has been retweeted !

File: package.json (hunk 1 above)
Comment (chimon2000) [reply package.json]: @kartsims I had to submit a pr to support the new rollup configuration, I will swap it out as that PR has been merged, and make the aforementioned changes.

Comment (kartsims) [reply package.json]: OK thanks for the feedback I'll update the post and create a `rollup` branch when it's ready

**Comment by chimon2000** [comment]
@kartsims all requested changes added!!!

**Comment by kartsims** [comment]
Yeah!

I tested it, removed a few more webpack dependencies, and got the following output : 

```
~/dev/vue-customelement-bundler ■ npm run dev

> vue-customelement-bundler@1.0.0 dev /Users/simon/dev/vue-customelement-bundler
> rollup -c -w

⚠   'default' is imported from external module 'fs' but never used

bundling...
⚠   Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
https://github.com/rollup/rollup/wiki/Troubleshooting#avoiding-eval
node_modules/vue-custom-element/dist/vue-custom-element.esm.js (23:4)
21:   if (typeof Symbol === 'undefined' || typeof Reflect === 'undefined') { return false; }
22:   try {
23:     eval('class Foo {}');
        ^
24:   } catch (e) {
25:     return false;

⚠   No format option was supplied – defaulting to 'es'
https://github.com/rollup/rollup/wiki/JavaScript-API#format

bundled in 3871ms. Watching for changes...
```

Any idea where these warnings come from ? I am merging the PR anyway (to a `rollup` branch) but that would be less scary without those warnings ;)

**Comment by kartsims** [comment]
What I did :

- Update the post (should be online in a few minutes)
- Created the rollup branch based on yours, minus a few remaining webpack dependencies
- Added reference to the branch in the master's README.md

The final gzipped file is 25% lighter, that is something ! Thanks again :)

**Pull request closed without merging**